spec/app/domain/commands/authentication/issue_token_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+RSpec.describe('Authentication::AuthnOidc::V2::Commands::ListProviders') do
+
+  let(:issue_token)  do
+    Commands::Authentication::IssueToken.new()
+  end
+
+  def create_account
+    Account.create("test")
+  end
+
+  describe('#call') do
+    context 'When you issue token for an existing account' do
```

Comment:
Same comment as below.  All contexts which start with `"when you issue token"` should go into a context with the more specific contexts inside.